CovidWatch/.idea/.name

```diff
@@ -1 +0,0 @@
-COVIDWatch
```

Comment:
How does a `.name` file get created?

Reply:
.name will be created automatically by Android Studio when opening the project, usually based on the root directory name.

I would commit it to avoid differences in /.idea/modules.xml file.

It should be the same as `rootProject.name` in  /settings.gradle

https://discuss.gradle.org/t/rootproject-name-in-settings-gradle-vs-projectname-in-build-gradle/5704/3